kernel/src/table_changes/physical_to_logical.rs

```diff
@@ -59,6 +60,7 @@ pub(crate) fn physical_to_logical_expr(
                 Ok(generated_column.unwrap_or_else(|| ColumnName::new([field_name]).into()))
             }
         })
+        .map(|expr| expr.map(Arc::new))
```

Comment:
This is called for each data file in the CDF code path. Also I'm not seeing any mention of transforms in CDF. We may have never updated it to handle transforms 😭 


Could you make a CDF issue. I can put together a CDF reads parent issue.

Reply:
started #1173